pkgdb2/api/admin.py

```diff
@@ -151,7 +151,7 @@ def api_admin_actions():
             status=status,
             count=True,
         )
-    except pkgdblib.PkgdbException, err:  # pragma: no cover
+    except pkgdblib.PkgdbException as err:  # pragma: no cover
```

Comment:
Here we use ``pkgdblib.PkgdbException`` but wasn't this the problem we were trying to address at first?

We adjusted ``api/acls.py`` to import the exception directly, should we adjust the others to behave in the same way?